lib/chef/compliance/default_attributes.rb

```diff
@@ -38,11 +38,12 @@ module Compliance
       # Allow for connections to HTTPS endpoints using self-signed ssl certificates.
       "insecure" => nil,
 
-      # Controls verbosity of Chef InSpec runner.
+      # Controls verbosity of Chef InSpec runner. See less output when true.
       "quiet" => true,
 
       # Chef Inspec Compliance profiles to be used for scan of node.
       # See README.md for details
```

Comment:
we might be able to add links to the docs page that will exist since we sort of know what the path will be

Reply:
This seems to refer to this section of the original README: https://github.com/chef-cookbooks/audit#configure-node

Reply:
Which I think corresponds to this (in preview that may be dated): https://deploy-preview-2870--chef-web-docs.netlify.app/chef_compliance_phase/#compliance-phase-configuration